service/src/main/resources/db/changesets/20220803_billing_profiles_managed_resource_group_id.yaml

```diff
@@ -0,0 +1,23 @@
+databaseChangeLog:
+  - changeSet:
+      id: billing_profile_managed_app_coords
+      author: aherbst
+      changes:
+        - addColumn:
+            tableName: billing_profile
+            columns:
+              - column:
+                  name: managed_resource_group_id
+                  type: text
+                  constraints:
+                    nullable: true
+                  remarks: |
+                    Azure Managed Resource Group ID for the Terra deployment
+        - dropColumn:
```

Comment:
Are there existing records in the DB to worry about, which will not have a `managed_resource_group_id` populated? Or is it empty/do we not care.

Reply:
It is empty, we do not care.